recipes/jaraco.logging/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jaraco.logging" %}
+{% set version = "1.5" %}
+{% set compress_type = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "05c8922640cd22e7c6cb71068ef494a227b61ae8a0cd43b28ab1fa08ef4f7cd3" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm >=1.15.0
+
+  run:
+    - python
+    - tempora
+    - six
+
+test:
+  imports:
+    - jaraco
+
+about:
+  home: https://github.com/jaraco/jaraco.logging
+  # license_file: jaraco doesn't include license files - see https://github.com/jaraco/skeleton/issues/1
```

Comment:
My only issue with using this for the Jaraco files is that there's no actual license file here; it's only included in the metadata. I can take a vanilla copy of the MIT License and throw that in, but that doesn't seem like it's going to be accurate - there's at least a copyright field that needs to be filled out.